### PR TITLE
workflows: configurable plot/refextract timeouts

### PR DESCRIPTION
**Pull request by michamos**
This also increases the default refextract timeout to 10 min, as the
current timeout is too short.

Signed-off-by: Micha Moskovic <michamos@gmail.com>

## Checklist:

- [ ] I have all the information that I need (if not, move to `RFC` and look for it).
- [ ] I linked the related issue(s) in the corresponding commit logs.
- [ ] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [ ] My code follows the code style of this project.
- [ ] I've added any new docs if API/utils methods were added.
- [ ] I have updated the existing documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: inspirehep/modules/workflows/tasks/arxiv.py [hunk 1]
```diff
@@ -62,6 +62,8 @@
     re.DOTALL)
 NO_PDF_ON_ARXIV = 'The author has provided no source to generate PDF, and no PDF.'
 
+PLOTEXTRACT_TIMEOUT = current_app.config.get('WORKFLOWS_PLOTEXTRACT_TIMEOUT', 5 * 60)
```
Comment (david-caro) [inline inspirehep/modules/workflows/tasks/arxiv.py]: you don't seem to use this anywhere no?

Comment (michamos) [reply inspirehep/modules/workflows/tasks/arxiv.py]: Good catch :sweat_smile:

File: tests/integration/workflows/test_arxiv_workflow.py [hunk 2]
```diff
@@ -54,10 +54,66 @@
...
+    assert pub_info
+    assert pub_info[0]
+    assert pub_info[0].get('year') == 2014
+    assert pub_info[0].get('journal_title') == "J. Math. Phys."
+
+    # A prediction should have been made
+    prediction = obj.extra_data.get("relevance_prediction")
+    assert prediction
+    assert prediction['decision'] == 'Non-CORE'
+    assert prediction['scores']['Non-CORE'] == 0.8358207729691823
```
Comment (szymonlopaciuk) [inline tests/integration/workflows/test_arxiv_workflow.py]: I'm not sure if it's the case, but could comparing floating-point numbers for equality here get flaky?

**Comment by ammirate** [comment]
ci merge please

**Pull request merged**